tracing/src/main/java/io/airlift/tracing/OpenTelemetryModule.java

```diff
@@ -110,4 +111,25 @@ private static String hostArch()
             default -> null;
         };
     }
+
+    private static class BatchSpanExporterProvider
```

Comment:
I would call this the `OltpGrpcSpanExporterProvider` since that is more important than the "batch" aspect (which is some local buffering and is effectively an implementation detail)